jsonpickle/pickler.py

```diff
@@ -251,7 +251,7 @@ def _flatten_bytestring(self, obj):
                 return obj.decode('utf-8')
             except:
                 pass
-        return {tags.B64: base64.encodestring(obj).decode('utf-8')}
+        return {tags.B64: base64.encodebytes(obj).decode('utf-8')}
```

Comment:
`base64.encodebytes()` does not exist in python2.  Is the end result different?  My naive testing showed that they're interchangeable, but I may be missing something.

I think you should be able to turn on the travis CI support on your clone and it should run the tests on both python2 and python3 in case that helps run the python2 tests without needing to install it locally.

Here are a few of the test failures when running on python2.7

```
ERROR: test handling of non-c-contiguous memory layout
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/numpy_test.py", line 181, in test_transpose
    _a, _b = self.roundtrip([a, b])
  [...]
  File "jsonpickle/pickler.py", line 284, in _flatten_obj_instance
    return handler(self).flatten(obj, data)
  File "jsonpickle/ext/numpy.py", line 221, in flatten
    if not obj.data.c_contiguous:
AttributeError: 'buffer' object has no attribute 'c_contiguous'
```

This failure seems a little surprising. The same test passes in a python3.4 virtualenv. Same line of code, different error message.
```
ERROR: test that cases with non-standard strides and offsets work correctly
----------------------------------------------------------------------
[...]
    return handler(self).flatten(obj, data)
  File "jsonpickle/ext/numpy.py", line 221, in flatten
    if not obj.data.c_contiguous:
AttributeError: cannot get single-segment buffer for discontiguous array
```

A couple of the tests also run into `AttributeError: 'module' object has no attribute 'encodebytes'` as mentioned above.

This last one is also surprising, though maybe not so much because the test *is* called "test_weird_arrays", so perhaps it's unsurprising? ;-)

```
======================================================================
FAIL: test behavior with memoryviews which are not ndarrays
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/numpy_test.py", line 224, in test_buffer
    self.assertEqual(len(w), 1)
AssertionError: 0 != 1
  
======================================================================
FAIL: test that we disallow serialization of references to arrays that do not effectively own their memory
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/numpy_test.py", line 165, in test_weird_arrays
    self.assertEqual(len(w), 1)
AssertionError: 0 != 1
```